src/plugins/intel_gpu/tests/unit/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -3158,6 +3268,24 @@ TEST_F(fully_connected_gpu_tests, compressed_int4_scale_b1g128) {
     this->test_compressed_int4_scale(false, false, 1, 128);
 }
 
+TEST_F(fully_connected_gpu_tests, compressed_int4_scale_dyn_quan_single_batch) {
+    this->test_compressed_int4_scale_dyn_quan(false, false);
+}
+
+TEST_F(fully_connected_gpu_tests, compressed_int4_scale_dyn_quan) {
+    this->test_compressed_int4_scale_dyn_quan(false, false, 512);
+}
+
+TEST_F(fully_connected_gpu_tests, compressed_int4_scale_dyn_quan_dynamic_single_batch) {
+    this->test_compressed_int4_scale_dyn_quan(false, true, 1);
+}
+
+
+TEST_F(fully_connected_gpu_tests, compressed_int4_scale_dyn_quan_dynamic) {
+    this->test_compressed_int4_scale_dyn_quan(false, true, 512);
+}
```

Comment:
could you also test for unaligned test?

Reply:
Added compressed_int4_scale_dyn_quan_unaligned and compressed_int4_scale_dyn_quan_dynamic_unaligned.